csvcubed/tests/test-cases/cli/build/cube_data_config_ok.csv

```diff
@@ -0,0 +1,3 @@
+Dim-0,Dim-1,Dim-2,Attr-1,Amount,Measure,Units
```

Comment:
Why have all of these test-case files been added in this PR? Are they being used anywhere?

Reply:
These files were being used by the test_build_friendly_error_msgs.py unit tests which I failed to commit.
I've now cleaned out a few files and updated the file names to match the exception being tested for and moved them
to tests/test-cases/readers/cube-config/v1.0/error_mappings folder.